### ISSUE TITLE
Checking CORSIKA and sim_telarray output of integration tests

### ISSUE THREAD

**Issue by GernotMaier** [gammasim/simtools#882]
Most integration tests are confirming at this point mostly that a simtools has been used without any errors.

For tests using CORSIKA or sim_telarray, we should add:

- test existence of expected output data and log files
- test log files for error messages

**Comment by VictorBarbosaMartins** [comment]
This was being addressed here: https://github.com/gammasim/simtools/pull/875

**Comment by GernotMaier** [comment]
Only partly. We should also have a look into the corsika / sim_telarray log files.

**Comment by VictorBarbosaMartins** [comment]
yes, but instead of overcomplicating it, I think we should fix what is already not working in https://github.com/gammasim/simtools/pull/875

**Comment by VictorBarbosaMartins** [comment]
Unless we still want to include testing the messages in the log files...

**Comment by GernotMaier** [comment]
I think we need to look inside the log files and check for errors. I have seen cases where corsika showed a segmentation fault but the integration test reported success.

**Comment by GernotMaier** [comment]
@orelgueta - maybe you can indicate if there are keywords missing to find failed or problematic CORSIKA or sim-telarray jobs. This is what we are testing right now:

```
ERROR_PATTERNS = [
    re.compile(r"error", re.IGNORECASE),
    re.compile(r"exception", re.IGNORECASE),
    re.compile(r"traceback", re.IGNORECASE),
    re.compile(r"\b(failed to|has failed)\b", re.IGNORECASE),
    re.compile(r"runtime\s*warning", re.IGNORECASE),
    re.compile(r"segmentation fault", re.IGNORECASE),
]
```

see simtools.testing.log_inspector

I would otherwise suggest to close this issue, as PR #1674 addressed the problem.

**Comment by orelgueta** [comment]
There's no way I can find anything missing off the top of my head or even by running CORSIKA multiple times with various errors in configuration. I suggest we close this issue and live with the current log inspector. If we run into other cases not covered there, we make sure to add them.
